documentation/eleccion_imagen.md

```diff
@@ -0,0 +1,39 @@
+# Decisión de la imagen elegida para el contenedor
+
+Antes de establecer ningún criterio es necesario tener en cuenta que elementos son necesarios para el proyecto. El primero es que se está trabajando con Go en linux, ningún biblioteca que no es está en la standard library es usada por lo que no es necesario instalación adicional en ese aspecto además de la propia de Go. Luego una instalación necesaria es la del gestor de tareas que en este caso es task (https://taskfile.dev/installation/).
+
+## Criterios de elección
+
+- En este caso vamos a tomar de referencia las recomendaciones en las buenas prácticas de docker (https://docs.docker.com/build/building/best-practices/) como criterios para la selección de la imagen para el contenedor. En base a esto se destacan se dice "When building your own image from a Dockerfile, ensure you choose a minimal base image that matches your requirements". Bajo esta idea se favorecerán aquellas imágenes que tengan lo mínimo para funcionar y por ende tengan un tamaño menor. El tamaño será tomado en cuenta una vez sean instalados los elementos necesarios establecidos anteriormente y la imagen sea construida.  
+
+- En caso de que una herramienta se encuentre en mala salud o abandona será descartada. 
```

Comment:
A parte de lo que te ha comentado mi compañero Mario de que es algo un poco obvio, me he fijado que en la descripción de las herramientas, no mencionas nada sobre la salud de cada una de las mismas, ni dices nada sobre la salud de la herramienta elegida, si no que solo se habla sobre el criterio del tamaño, con ello, supongo que o bien debería de mencionar algo en cada paquete o bien suprimir este criterio.

Reply:
Tenéis razón ambos. Ese criterio solo fue usado para cicle-ci ya que estaba archivada pero es verdad que el resto de herramientas no era algo relevante ya que todas están en buena salud. El criterio fue suprmidio en 096564027b2a177a253ae801a91ef77e80229d4a